React/finger-exercises/ecmascript/02 - promises/index.js

```diff
@@ -0,0 +1,17 @@
+// Hint: use setInterval, create a new Promise and measure time with Date.now()
+
+export function delay(time) {
+  return new Promise((resolve, reject) => {
+    if (time >= 1000) {
+      throw new Error("This time is too much !");
+    } else {
+      setTimeout(() => {
+        resolve(time);
+      }, time);
+    }
+  });
+}
+
+export async function asyncDelay(time) {
```

Comment:
You don't have to make this an async function since the asynchronous process is done in the function your returning.